src/wp-includes/cache-compat.php

```diff
@@ -141,3 +141,29 @@ function wp_cache_flush_runtime() {
 		return wp_using_ext_object_cache() ? false : wp_cache_flush();
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_flush_group' ) ) :
+	/**
+	 * Removes all cache items in a group.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @see WP_Object_Cache::flush_group()
+	 * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+	 *
+	 * @param string|array $group name(s) of group to remove from cache.
+	 *
+	 * @return bool|Array True or array of bool in array passed on success, false on failure group not found.
+	 */
+	function wp_cache_flush_group( $group ) {
+		global $wp_object_cache;
+
+		if ( $wp_object_cache && method_exists( $wp_object_cache, 'flush_group' ) ) {
+			return $wp_object_cache->flush_group( $group );
+		}
+
+		_doing_it_wrong( __FUNCTION__, __( 'Your implementation of object caching does not support flush by group.' ), '6.0.0' );
+
+		return wp_cache_flush();
```

Comment:
Check for `WP_OBJECT_CACHE_SUPPORTS_GROUP_FLUSH`.

```suggestion
		if ( WP_OBJECT_CACHE_SUPPORTS_GROUP_FLUSH ) {
			return $wp_object_cache->flush_group( $group );
		}

		$error = new WP_Error( 'unsupported', __( 'Your object cache implementation does not support flushing individual groups.' ) );

		_doing_it_wrong( __FUNCTION__, $error->get_error_message() , '6.0.0' );

		return $error;
```